### PR TITLE
added russell to the project roles

### PR DESCRIPTION
**Pull request by torahhorse**
I think russell would be a good fit for this project as temporary lawyer

### CODE REVIEW

File: Project Roles [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Recruiter - Torahhorse
 Office Manager - Jacob
 Fighting Game Consultant - Yuliy
+Team Lawyer (temporary) - Russell
```
Comment (jacobknipfing) [inline Project Roles:4]: did you hear back from your aunt yet?

Comment (torahhorse) [inline Project Roles:4]: No

Comment (torahhorse) [inline Project Roles:4]: That is why I suggest Russell as temporary

Comment (jacobknipfing) [inline Project Roles:4]: I already merged the pull request

**Pull request merged**